SPECS/nginx/nginx.spec

```diff
@@ -59,6 +59,7 @@ sh configure \
     --sbin-path=%{_sbindir}/nginx                 \
     --user=%{nginx_user} \
     --with-http_auth_request_module \
+    --with-http_gunzip_module \
```

Comment:
Pardon my ignorance here. 
Does that mean that the nginx web server will always use rely on compression/decompression? or is that configurable? 
Mainly concerned about any performance regressions if any for this change. (since this is a commonly ised package as you may think)

Reply:
no problem :)

It won't always rely on compression/decompression, this simply enables the build of the gunzip module (see: https://nginx.org/en/docs/http/ngx_http_gunzip_module.html)

Here's a super simplistic example of how this module can be used in combination with other modules already built with the current mariner nginx config:
```
http {
       ...
        server {
         ...
         gzip on;
         server_name _;
         location / {
           gzip_static always;
           gunzip on; #<<-- what this updated rpm would support
         }
        }
}
```

Here any static file eg `hello.gz` can be requested by simple clients eg `wget` buy requesting `/hello`, nginx will transparently serve the decompressed contents of hello.gz.

This build configuration is very common amongst distros, eg:
 * https://src.fedoraproject.org/rpms/nginx//blob/rawhide/f/nginx.spec#_316
 * http://hg.nginx.org/pkg-oss/file/tip/rpm/SPECS/Makefile#l77
 
There is no performance impact if the module is not used/specified in the configuration - this is not a breaking change. If the module is used and made use of the CPU and memory consumption does increase over simple serving of static files, though this is offset by significant savings in on disk space sondumed by the content.

Reply:
Thanks for the detailed explanation, Pete.